Cannot find module 'defaults'

**Steps to reproduce**

1. Using the following example config `.releaserc`: 

```
{
  "tagFormat": "${version}",
  "branch": "master",
  "verifyConditions": [
    "@semantic-release/changelog",
    "@semantic-release/git",
    "@qiwi/semantic-release-gh-pages-plugin"
  ],
  "publish": [
    "@semantic-release/github",
    {
      "path": "@qiwi/semantic-release-gh-pages-plugin",
      "msg": "deploy: new gh-pages release",
      "src": "build",
      "branch": "gh-pages"
    }
  ]
}
```
2. run `npx semantic-release`
3. See error below


**Additional information**

```
An error occurred while running semantic-release: { Error: Cannot find module 'defaults'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (<>/node_modules/@qiwi/semantic-release-gh-pages-plugin/lib/es5/index.js:10:22)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3) code: 'MODULE_NOT_FOUND' }
{ Error: Cannot find module 'defaults'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (<>node_modules/@qiwi/semantic-release-gh-pages-plugin/lib/es5/index.js:10:22)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3) code: 'MODULE_NOT_FOUND' }%
```

--- a/src/main/index.ts
+++ b/src/main/index.ts
@@ -5,7 +5,7 @@ import { publish as ghpagesPublish } from './ghpages'
 import { render } from './tpl'
 import { isEqual } from 'lodash'
 
-export * from 'defaults'
+export * from './defaults'
 
 let _config: any
 

